ga4gh/cli.py

```diff
@@ -1496,6 +1498,29 @@ def configtest_main(parser=None):
 ##############################################################################
 
 
+def getNameFromPath(filePath):
+    """
+    Returns the filename of the specified path without its extensions.
+    This is usually how we derive the default name for a given object.
+    """
+    if len(filePath) == 0:
+        raise ValueError("Cannot have empty path for name")
+    fileName = os.path.split(filePath)[1]
+    # We need to handle things like .fa.gz, so we can't use
+    # os.path.splitext
+    ret = fileName.split(".")[0]
+    assert ret != ""
```

Comment:
The assert? I think this classifies as a programming error if we accept the input and the output is an empty string. This is just trying to be defensive, putting in asserts for stuff that shouldn't happen, but might.